gc/stats/GlobalGCStats.hpp

```diff
@@ -96,10 +94,10 @@ class MM_GlobalGCStats {
 #endif /* OMR_GC_MODRON_COMPACTION */
 		, fixHeapForWalkReason(FIXUP_NONE)
 		, fixHeapForWalkTime(0)
+		, fixHeapForWalkObjectCount(0)
 		, markStats()
 		, classUnloadStats()
-		, metronomeStats()
-		, finalizableCount(0) {};
+		, metronomeStats() {};
```

Comment:
while touching this this line into 2 (try to see other constructor, but I believe curly braces should be in a separate line)

Reply:
Yes I believe so, will do this, and will also fix the comma location for all lines to be consistent with how it is done more or less everywhere else in the code.